src/backend/core/api/viewsets.py

```diff
@@ -186,7 +186,7 @@ class UserViewSet(
     """
 
     permission_classes = [permissions.IsSelf]
-    queryset = models.User.objects.all()
+    queryset = models.User.objects.all().order_by("-created_at")
```

Comment:
I wonder, if by default we should not define an ordering field in the `BaseModel`, might solve all warnings

Reply:
good idea since we have this field on all models!

Reply:
will do next